README.md

```diff
@@ -70,10 +83,9 @@ Add a `.emdaer/README.emdaer.md` file:
 
 ```md
 # <!--emdaer-p
-  - '@emdaer/plugin-value-from-package'
-  - value: name
--->
 
+* '@emdaer/plugin-value-from-package'
+* value: name -->
```

Comment:
This is problematic...

Reply:
`<!-- prettier-ignore -->` can take care of this since its an edge case. However, it may be worth opening an issue in prettier.